src/OneExplorer.ts

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+enum NodeType{
+  directory,
+  model,
+  config
+}
+
+interface Node {
+  type: NodeType, name: string, childs: Node[], uri: vscode.Uri
+}
+
+export class OneNode extends vscode.TreeItem {
+  constructor(
+      public readonly label: string,
```

Comment:
```suggestion
      public readonly tooltip: string,
```
`label` can mean many things so it's hard to read code to figure out what's the meaning of label here. But we know what `tooltip` is.

Reply:
@hyunsik-yoon 
This `label` variable is not same with `tooltip` string. The tooltip string can be changed regardless label string. So I prefer to keep `label` variable. Instead, as you said, it is hard to figure out the meaning, so the name can be changed. How about the name variable? This string means the file or directory name.

Reply:
> @hyunsik-yoon
> 
> > label can mean many things so it's hard to read code to figure out what's the meaning of label here. But we know what tooltip is.
> 
> the term 'label' is a member of vscode.TreeItem. It's different from tooltip. tooltip is another member of it. `tooltip` defines what to show on a tooltip, I think `label` has different purpose, for example showing the asset label when uri parsing is failed or so.

Reply:
I will remove either of `toNode` function or OneNode contstructor, they do the same-purposed jobs in different functions and making confusion.